lib/internal/bootstrap/loaders.js

```diff
@@ -95,7 +95,8 @@ const internalBindingWhitelist = new SafeSet([
   'util',
   'uv',
   'v8',
-  'zlib'
+  'zlib',
+  'utilities'
```

Comment:
this list lets them be accessed by `process.binding`, it definitely shouldn't be accessible like that.

Reply:
Jm, I understood something different: " propose we try creating a new, internal namespace (that is, a namespace that's not in the whitelist here) and try moving everything away to that internal namespace to reduce the abuse." (https://github.com/nodejs/node/issues/30884#issue-535804474)

Reply:
How it should be?